test/rules/no-default-export/test.ts.lint

```diff
@@ -0,0 +1,4 @@
+export const default = 'VALID';
```

Comment:
it is technically the same, but expressive instead of implicit. I feel like it should be allowed because it is not using the default export keyword.